.github/workflows/ubuntu_24.yml

```diff
@@ -17,7 +17,6 @@ permissions: read-all
 
 env:
   PIP_CACHE_PATH: /mount/caches/pip/linux
-  PYTHON_VERSION: '3.11'
```

Comment:
Unfortunately, we can't use `${{ env.PYTHON_VERSION }}` as an input for a reusable workflow as the `env` context is not propagated to the reusable jobs, we can do a workaround like in [1](https://github.com/orgs/community/discussions/26671) but currently we only have one variable needing to be passed so we might have it as it is now.